scripts/mbedtls_framework/psa_compliance.py

```diff
@@ -105,9 +105,9 @@ def test_compliance(library_build_dir: str,
             '^TEST RESULT: (?P<test_result>FAILED|PASSED)'
         )
         test = -1
-        unexpected_successes = expected_failures.copy()
-        expected_failures.clear()
-        unexpected_failures = [] # type: List[int]
+        unexpected_successes = set(expected_failures)
+        seen_expected_failures = set()
+        unexpected_failures = set()
```

Comment:
For my own education: how comes with a list we needed to give pylint a type hint (a list of what) but with a set apparently we don't?

Reply:
Thanks!